cmd/githubrunnerscalesetlistener/metrics.go

```diff
@@ -0,0 +1,381 @@
+package main
+
+import (
+	"strconv"
+
+	"github.com/actions/actions-runner-controller/github/actions"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// label names
+const (
+	labelKeyRunnerScaleSetName            = "name"
```

Comment:
`labelKeyRunnerScaleSetName` and `labelKeyAutoScalingRunnerSetName` might confuse customers?